rpc/blockchain.go

```diff
@@ -817,15 +817,26 @@ func (s *PublicBlockchainService) GetStakingNetworkInfo(
 	})
 }
 
+const (
+	// If peer have block height difference smaller or equal to 10 blocks, the node is considered inSync
+	inSyncTolerance = 10
```

Comment:
if we tolerant 10 blocks, there could be inconsistent results to the RPC client.

Reply:
What do you man by RPC client?

Reply:
I mean dApp that queries the blockchain state.  It may query the node that is 10 blocks old which may be inconsistent with the results that returned by a node that is sync'ed to the latest block.